docker/package/packages.py

```diff
@@ -441,8 +441,13 @@ def mk_node_unit(
         }
     )
 
-sapling_package = TezosSaplingParamsPackage(
-    meta=packages_meta, params_revision="95911b0639ff01807b8fd7b9e36d508e657d80a8"
+packages.append(
```

Comment:
The problem is that this packages is now built by default, which means that it'll be uploaded to the Copr/Launchpad on the next `v*-1` release, while we don't really want to update it

Reply:
What about adding `--release` flag and filter it out in `docker-tezos-packages.py`?

Reply:
There was a flag that explicitly enables this package building, I liked that approach